src/cpp/fastdds/builtin/type_lookup_service/detail/rpc_types.idl

```diff
@@ -1,7 +1,9 @@
+//This directive is not suported.
```

Comment:
```suggestion
//This directive is not supported yet.
```

Reply:
Done in [fc1a5d9](https://github.com/eProsima/Fast-DDS/pull/4054/commits/fc1a5d981bdf6528b1854832b33b401eedabe678)